bin/install

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 
+basename=`dirname $0`
+cd $basename/..
```

Comment:
please quote "$expresssion"s.
and, eek doing a cd followed by rm -Rf is not particularly friends.